L1Trigger/TrackerDTC/interface/Setup.h

```diff
@@ -135,6 +175,20 @@ namespace trackerDTC {
     double basePhi() const { return basePhi_; }
     // number of padded 0s in output data format
     int dtcNumUnusedBits() const { return dtcNumUnusedBits_; }
+    // outer radius of outer tracker in cm
+    double outerRadius() const { return outerRadius_; }
+    // inner radius of outer tracker in cm
+    double innerRadius() const { return innerRadius_; }
+    // range of stub z residual w.r.t. sector center which needs to be covered
+    double neededRangeChiZ() const { return neededRangeChiZ_; }
+    // half length of outer tracker in cm
+    double halfLength() const { return halfLength_; }
+    // max strip/pixel length of outer tracker sensors in cm
+    double maxLength() const { return maxLength_; }
+    //
```

Comment:
Add comment explaining what this is. e.g. "In tilted barrel, grad*|z|/r+int approximates  |cos(theta) - tiltAngle|/sin(theta)|".
Does analysis code provide histogram to fit the slope & intercept? e.g. /home/ppd/tomalin/UK_TrackTrig/hybrid11/CMSSW_11_2_0_pre6/src//L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc